src/main/java/seedu/address/ui/AppointmentCard.java

```diff
@@ -73,7 +73,7 @@ public AppointmentCard(Appointment appointment) {
 
         // This line of code ensures that if it's not visible, it doesn't take up any space in the layout
         feedbackScore.managedProperty().bind(feedbackScore.visibleProperty());
-        if (appointment.getFeedbackScore() != null) {
+        if (appointment.getFeedbackScore().feedbackScore != null) {
```

Comment:
Naming scheme a bit funny maybe can fix in v1.4

Reply:
Yeah I agree it's funny. Not sure how else to name it, but can KIV